content/en/post/tem-hca/index.md

```diff
@@ -0,0 +1,45 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "Mr. Thonglek's talk at IEICE-EE 2021"
+summary: ""
+authors: ["kundjanasith-thonglek"]
+tags: ["Machine Learning"]
+categories: []
+date: 2021-05-27T10:29:27+09:00
+
+# Optional external URL for project (replaces project detail page).
+external_link: ""
+
+# Featured image
+# To use, add an image named `featured.jpg/png` to your page's folder.
+# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
+image:
+  caption: ""
+  focal_point: ""
+  preview_only: false
+
+# Custom links (optional).
+#   Uncomment and edit lines below to show custom links.
+# links:
+# - name: Follow
+#   url: https://twitter.com
+#   icon_pack: fab
+#   icon: twitter
+
+url_code: ""
+url_pdf: ""
+url_slides: ""
+url_video: ""
+
+# Slides (optional).
+#   Associate this project with Markdown slides.
+#   Simply enter your slide deck's filename without extension.
+#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
+#   Otherwise, set `slides = ""`.
+slides: ""
+---
+
+Mr. Kundjanasith Thonglek (Tem), a first-year doctoral student at SDLab presented the follwing paper at the [Technical Meeting on Home and Consumer Appliances (IEICE-EE) 2021](https://www.ieice.org/ken/program/index.php?mode=program&tgs_regid=da7e1616e5f2257e688d8bf46f549def5a7a9e50dfa313bfb258a79e10aed93a&tgid=IEE-HCA&layout=&lang=eng)
```

Comment:
"Technical Meeting on Home and Consumer Appliances (IEICE-EE) 2021"
-> 
"Technical Committee on Energy Engineering in Electronics and Communications (IEICE-EE), 2021 May"